File: Contador/app/src/main/java/com/example/contador/MainActivity.java
package com.example.contador;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends Activity {
    //Variable publico
    public int contador;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //Inicializar la variable a cero
        //Para que no guarde ningun datos
        contador=0;
    }


    // Creao loa metodos para realizar las funciones del boton
    public void  IncrementarContador(View vista){
        contador++;
        mostarResultado();
    }

    public void resetearContador(View vista){

        contador=0;
        mostarResultado();

    }

    public  void desecrementacontador(View vista){
        contador--;
        mostarResultado();
    }

    public void mostarResultado(){
        TextView textoResultado=(TextView) findViewById(R.id.contaodorNumeros);

        textoResultado.setText(" "+contador);

    }


}

File: Udacity_First/app/src/main/java/com/example/udacity_first/MainActivity.kt
package com.example.udacity_first

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var rollButton: Button = findViewById(R.id.roll_button)
        rollButton.text =  "Let's go it"
    }
}
